lib/config.js

```diff
@@ -68,6 +68,7 @@ config.cronIntervalWorkspaceTimeoutWarnSec = 60;
 config.cronIntervalWorkspaceHostLoadsSec = 10;
 config.cronIntervalWorkspaceHostTransitionsSec = 10;
 config.cronDailySec = 8 * 60 * 60;
+config.nodeMetricsIntervalSec = 5;
```

Comment:
Should we make this default to a value (maybe 0) that causes this to be inactive? Even if we default to active, I think it would be good to have a way to deactivate it via a config option (either treat 0 as inactive, or add an explicit boolean flag).

Reply:
How about using `null` to disable?